ereuse_devicehub/resources/lot/models.py

```diff
@@ -103,6 +104,18 @@ def descendants(self):
     def is_temporary(self):
         return False if self.trade else True
 
+    @property
+    def is_incominig(self):
```

Comment:
Fix typo in all the code
```suggestion
    def is_incoming(self):
```

Reply:
done